adminSDK/alertcenter/quickstart/src/main/java/AdminSDKAlertCenterQuickstart.java

```diff
@@ -0,0 +1,106 @@
+public class AdminSDKAlertCenterQuickstart {
+
+  private static final String APPLICATION_NAME = "Google Admin SDK Alert Center API Java Quickstart";
+  private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+  /**
+   * Global instance of the scopes required by this quickstart. If modifying these scopes, delete
+   * your previously saved tokens/ folder.
+   */
+  private static final List<String> SCOPES = Collections
+      .singletonList("https://www.googleapis.com/auth/apps.alerts");
+  private static final String CREDENTIALS_FILE_PATH = "/credentials.json";
+
+
+  /**
+   * Creates an authorized Credentials object.
+   *
+   * @return An authorized Credentials object.
+   * @throws IOException If the credentials.json file cannot be found.
+   */
+  private static Credentials getCredentials(String delegatedAdmin) throws IOException {
+    // [START admin_sdk_alertcenter_get_credentials]
+    InputStream in = AdminSDKAlertCenterQuickstart.class.getResourceAsStream(CREDENTIALS_FILE_PATH);
+    if (in == null) {
+      throw new IOException("Credential file was not found");
+    }
+    GoogleCredentials credentials = ServiceAccountCredentials
+        .fromStream(in)
+        .createDelegated(delegatedAdmin)
+        .createScoped(SCOPES);
+    // [END admin_sdk_alertcenter_get_credentials]
+    return credentials;
+  }
+
+  public static void main(String... args) throws IOException, GeneralSecurityException {
+    // [START admin_sdk_alertcenter_create_client]
+    String delegatedAdmin = "admin@xxx.com";
+    NetHttpTransport transport = GoogleNetHttpTransport.newTrustedTransport();
+    AlertCenter service = new AlertCenter.Builder(transport, JSON_FACTORY,
+        new HttpCredentialsAdapter(getCredentials(delegatedAdmin)))
+        .setApplicationName(APPLICATION_NAME)
+        .build();
+    // [END admin_sdk_alertcenter_create_client]
+
+    // [START admin_sdk_alertcenter_list_alerts]
+    ListAlertsResponse listResponse = service.alerts().list().setPageSize(20).execute();
+    while (listResponse.getAlerts() != null) {
+      for (Alert alert : listResponse.getAlerts()) {
+        System.out.println(alert);
+      }
+      if (listResponse.getNextPageToken() == null || listResponse.getNextPageToken().isEmpty()) {
+        break;
+      }
+      listResponse = service.alerts().list().setPageToken(listResponse.getNextPageToken())
+          .setPageSize(20).execute();
+    }
+    // [END admin_sdk_alertcenter_list_alerts]
+
+
+    listResponse = service.alerts().list().setPageSize(20).execute();
+    if (listResponse == null || listResponse.isEmpty()) {
+      System.out.println("No alerts");
+    }
+    String alertId = listResponse.getAlerts().get(0).getAlertId();
+    // [START admin_sdk_alertcenter_provide_feedback]
+    AlertFeedback newFeedback = new AlertFeedback();
+    newFeedback.setType("VERY_USEFUL");
+    AlertFeedback feedback = service.alerts().feedback().create(alertId, newFeedback).execute();
```

Comment:
We prefer to keep quickstarts read-only whenever possible, to avoid changing production data accidentally. If you want to keep this sample code in the quickstart, I'd suggest moving it to a separate function, and include a commented-out call to that function, like:

```
// Uncomment the line below to set alert feedback.
// setAlertFeedback(service, alertId);
```